includes/template-library/sources/remote.php

```diff
@@ -210,6 +216,91 @@ public function get_data( array $args, $context = 'display' ) {
 		return $data;
 	}
 
+	/**
+	 * Get templates data from transient or by remote request.
+	 * The remote request will happen in one of those 3 conditions:
+	 * 1. Force update.
+	 * 2. The data saved in the transient is empty.
+	 * 3. The container feature has changed since the last data fetch.
+	 *
+	 * @param $force_update
+	 * @return array
+	 */
+	private function get_templates_data( $force_update ) {
+		$templates_data_cache_key = static::TEMPLATES_DATA_TRANSIENT_KEY_PREFIX . ELEMENTOR_VERSION;
```

Comment:
No need to use static when they are not static. Can use self::
But it does not really make any difference technically. Just for readability.